packages/cli/src/commands/setup/i18n/templates/fr.json.template

```diff
@@ -0,0 +1,11 @@
+{
+  "Welcome to RedwoodJS": "Bienvenu sur RedwoodJS",
+  "info": "Ceci est votre fichier de traduction",
+  "see": "https://www.i18next.com/translation-function/essentials",
+  "HomePage": {
+    "title": "Page d'accueil",
+    "info": "Touve moi dans",
```

Comment:
```suggestion
    "info": "Trouve moi dans",
```
^ i don't speak French but was this a typo?

Reply:
Yes is a typo! Thank you, I didn't see it!